pkg/apis/planetscale/v2/vitesskeyspace_types.go

```diff
@@ -401,13 +401,33 @@ type VitessKeyspacePartitioningStatus struct {
 	// possible that some shards in this partitioning are serving writes.
 	// Check the per-shard status for full details.
 	ServingWrites corev1.ConditionStatus `json:"servingWrites,omitempty"`
+	// DesiredTablets is the number of desired tablets. This is computed from
+	// information that's already available in the spec, but clients should
+	// use this value instead of trying to compute shard partitionings on their
+	// own.
+	DesiredTablets int32 `json:"desiredTablets,omitempty"`
+	// Tablets is the number of observed tablets. This could be higher or
+	// lower than desiredTablets if the state has not yet converged.
+	Tablets int32 `json:"tablets,omitempty"`
+	// ReadyTablets is the number of desired tablets that are Ready.
+	ReadyTablets int32 `json:"readyTablets,omitempty"`
+	// UpdatedTablets is the number of desired tablets that are up-to-date
+	// (have no pending changes).
+	UpdatedTablets int32 `json:"updatedTablets,omitempty"`
 }
 
 // NewVitessKeyspacePartitioningStatus creates a new status object with default values.
 func NewVitessKeyspacePartitioningStatus(partitioning *VitessKeyspacePartitioning) VitessKeyspacePartitioningStatus {
+	desiredTablets := int32(0)
```

Comment:
Nit: The zero value is 0, so this is fine right?
```suggestion
	var desiredTablets int32
```

Reply:
I was copying logic from another constructor function above in the same file. But I believe you're right on that.

Reply:
Updated here, and also updated the place I copied it from.

Reply:
Cool, thanks!